check_borg

```diff
@@ -84,21 +95,21 @@ sec_crit="$(date --date="${crit}" '+%s')"
 
 # check warning and critical values
 if check_range "${sec_crit}" 0:"${sec_warn}" ; then
-	# 
+	#
 	error "Warning value has to be a more recent timepoint than critical."
 fi
 
 # get unixtime of last backup
-sec_last="$(borg list --format '{time}{NUL}' "${repo}" | xargs '-0' '-I&' 'date' '--date=&' '+%s' | sort | tail -n 1)"
-last="$(date --date="@${sec_last}")"
+last="$(borg list --format '{time}' --sort-by timestamp --last 1 ${repo})"
+sec_last=$(date --date="${last}" +%s)
```

Comment:
Please quote the date output.

Thanks for asserting, that date should spit out unix seconds.